src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar primary-search-bar"
+      data-qa="big"
```

Comment:
You've used the class names `search-bar` and `primary-search-bar` here. Class names should be semantic and describe the meaning of the content, not the styles or tag names. Consider renaming `primary-search-bar` to something more descriptive of its function or content, for example `main-search-bar`.